src/Majima/task/TaskList.java

```diff
@@ -0,0 +1,46 @@
+package Majima.task;
+
+import Majima.MajimaException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+
+    private List<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public TaskList(List<Task> tasks) {
+        this.tasks = new ArrayList<>(tasks);
+    }
+
+    public void addTask(Task task) {
```

Comment:
consider putting javadocs for your public methods!

Reply:
Will do, in a later commit. Thanks!